Index: src/test/java/Amazon/TestAmazonBasicsPage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Amazon;\n\nimport base.TestBasePage;\n\npublic class TestAmazonBasicsPage extends TestBasePage {\n}\n
===================================================================
diff --git a/src/test/java/Amazon/TestAmazonBasicsPage.java b/src/test/java/Amazon/TestAmazonBasicsPage.java
--- a/src/test/java/Amazon/TestAmazonBasicsPage.java	
+++ b/src/test/java/Amazon/TestAmazonBasicsPage.java	
@@ -1,6 +1,50 @@
 package Amazon;
 
+
+
 import base.TestBasePage;
 
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
 public class TestAmazonBasicsPage extends TestBasePage {
+    @Test
+    public void testAmazonBasicsPageSearch(){
+        Homepage homepage = new Homepage();
+        AmazonBasicsPage amazonBasicsPage = homepage.navigateToAmazonBasicsPage();
+        String text = "pet food";
+        amazonBasicsPage.searchBar.sendKeys(text);
+        amazonBasicsPage.searchBar.click();
+
+        String expectedText = "pet food";
+
+        Assert.assertEquals(expectedText, text);
+    }
+    @Test
+    public void testNewArrivalsPage(){
+        Homepage homepage = new Homepage();
+        AmazonBasicsPage amazonBasicsPage = homepage.navigateToAmazonBasicsPage();
+        amazonBasicsPage.newArrivals.click();
+        String text = "organiger";
+        amazonBasicsPage.searchBar.sendKeys(text);
+        amazonBasicsPage.searchBar.click();
+
+        String expectedText = "organiger";
+
+        Assert.assertEquals(expectedText, text);
+    }
+    @Test
+    public void testOfficeSuppliesPage(){
+        Homepage homepage = new Homepage();
+        AmazonBasicsPage amazonBasicsPage = homepage.navigateToAmazonBasicsPage();
+        amazonBasicsPage.officeSupplies.click();
+        String text = "stappler";
+        amazonBasicsPage.searchBar.sendKeys(text);
+        amazonBasicsPage.searchBar.click();
+
+        String expectedText = "stappler";
+
+        Assert.assertEquals(expectedText, text);
+    }
+
 }
Index: src/main/java/Amazon/AmazonBasicsPage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Amazon;\n\nimport base.BasePage;\n\npublic class AmazonBasicsPage extends BasePage {\n}\n
===================================================================
diff --git a/src/main/java/Amazon/AmazonBasicsPage.java b/src/main/java/Amazon/AmazonBasicsPage.java
--- a/src/main/java/Amazon/AmazonBasicsPage.java	
+++ b/src/main/java/Amazon/AmazonBasicsPage.java	
@@ -1,6 +1,37 @@
 package Amazon;
 
 import base.BasePage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+
 
 public class AmazonBasicsPage extends BasePage {
-}
+    private static final String URL = "https://www.amazon.com/";
+    @FindBy(className = "style__navigation__FMWJE Header__navigationWithFollowButton__3F3fa Header__hasLogo__3osQs style__hasLogo__3Lfx0 style__hasFollowButton__enDxN")
+    public WebElement searchBar;
+
+    @FindBy(xpath = "//*[@id=\"k9rs2mhe3v\"]/div/div/div/a")
+    public WebElement newArrivals;
+
+    @FindBy(xpath = "//*[@id=\"uaxidqaeev\"]/div/div/div/a")
+    public WebElement officeSupplies;
+
+    @FindBy(xpath = "//*[@id=\"vpwtwtp1lu\"]//a")
+    public WebElement bedding;
+
+    @FindBy(xpath = "//*[@id=\"cxrj9a6ao3\"]/div/div/div/a")
+    public WebElement petSupplies;
+
+    public AmazonBasicsPage() {
+        webDriverWait.until(ExpectedConditions.urlContains(URL));
+        PageFactory.initElements(driver, this);
+    }
+
+//    public void selectDifferentOption(String visibleText) {
+//        selectFromDropdownByValue(dropdown, visibleText);
+//    }
+
+}
\ No newline at end of file
